wiki/Ranking_criteria/Simplified_ranking_criteria/en.md

```diff
@@ -0,0 +1,199 @@
+- **Don't include unused files in the map folder.**
+
+### Song
+
+- **Use `.mp3` or `.ogg` file formats.**
+  - At most 192 kbps for `.mp3` files.
+  - At most 208 kbps for `.ogg` files.
+  - At least 128 kbps for any filetype.
+- **One song file must be used for all difficulties.**
+- **Set a consistent preview point for all difficulties.**
+
+### Hitsounds
+
+::: Infobox
+![](img/delay.png "Hitsound file with delay")
+:::
+
+- **Hitsound files must be 25ms long and use `.wav` or `.ogg` file formats.**
+  - Do not use `.mp3`.
+  - **Exception:** Use [this file](https://up.ppy.sh/files/blank.wav) for silent hitsounds.
+- **Hitsounds must not be [delayed](img/delay.png).**
+
+### Background
+
+- **Every difficulty must have a background adhering to the following:**
+  - **Minimum width:** 160px
+  - **Minimum height:** 120px
+  - **Maximum width:** 2560px
+  - **Maximum height:** 1440px
+  - **Maximum file size:** 2.5MB
+- **Link the source of the map's background in the description.**
+
+### Video
+
+- **Video must adhere to the following:**
+  - **Maximum video resolution:** 1280x720
+  - **Video encoding:** H.264
+- **Audio track must be removed from video files.**
+
+## Mode-specific
+
+*For full [game mode](/wiki/Game_mode)-specific ranking criteria, see: [osu!](../osu!), [osu!taiko](../osu!taiko), [osu!catch](../osu!catch), and [osu!mania](../osu!mania)*
+
+### osu!
+
+::: Infobox
+![](img/offscreen.png "Slightly offscreen slider in 1280x960")
+:::
+
+- **No off-screen objects in 4:3 aspect ratios.**
+- **Auto must achieve bonus score on spinners.** They're too short otherwise.
+- **Use at least two combo colors.**
+
+### osu!taiko
+
+- **Adjust the background's vertical offset to fit the playfield.**
+- **Use `Omit first bar line` to remove any unfitting bar lines.**
+- **Avoid rapid kiai toggles.**
+- **On rhythms that are 1/4 or faster, use big notes only at the end of patterns.**
+- **Avoid slider velocity changes on lower difficulties.**
+- **Keep the base slider velocity at 1.4.**
+- **Each difficulty should follow its respective break length guidelines:**
+
+| Difficulty | Break | Chain length |
+| --: | :-: | :-: |
+| **Kantan** | ![3/1](img/taiko/kantan.png "3/1") | Break needed every 32-36 beats |
+| **Futsuu** | ![2/1](img/taiko/futsuu.png "2/1") | Break needed every 32-36 beats |
+| **Muzukashii** (option 1) | ![3/2](img/taiko/muzu1.png "3/2") | Break needed every 16-20 beats |
+| **Muzukashii** (option 2) | ![3 consecutive 1/1](img/taiko/muzu2.png "3 consecutive 1/1") | Break needed every 16-20 beats |
+| **Oni** | ![1/1](img/taiko/oni.png "1/1") | Break needed every 16-20 beats |
+
+### osu!catch
+
+- something
+
+### osu!mania
+
+- **[Spread](#spread) rules apply to each key mode separately.** For example, if you have 4K and 7K difficulties, you need two difficulty spreads.
```

Comment:
```suggestion
- **[Spread](#spread) rules apply to each key mode or [playstyle](/wiki/Ranking_criteria/osu!mania#common-terms) separately.** For example, if you have 4K and 7K difficulties, you need two difficulty spreads.
```

Worth clarifying these are two similar yet distinct ways for Spread rules to be split between sub-sets.